src/services/contact.js

```diff
@@ -0,0 +1,11 @@
+import emailjs from '@emailjs/browser';
+
+const contactUs = (e, templateParams) => {
```

Comment:
could you rename this to something relevant, like mailSender or something.

Did you test the function with a basic input you created ? Does it work properly ?

Reply:
yes, I created a form and tested the function its works properly, and I will change the name